Jenkinsfile

```diff
@@ -40,12 +45,8 @@ devToolsProject.run(
   deployWhen: { return runTheBuilds.isPushTo(['master']) || env.FORCE_DEPLOY == 'true' },
   deploy: { data ->
     data['dtrImage'].push()
-    data['dtrImage'].deploy(
-      '8000',
-      '-v jenkins-nodes:/jenkins_nodes',
-      '--target-port 9100 --target-port 9323' +
-        " --url ${env.JENKINS_URL} /jenkins_nodes/output.json",
-    )
+    String cliArgs = encryptedFile.read(path: 'cliArgs.enc', credentialsId: CREDENTIAL_ID)
```

Comment:
A bit petty, but I'm not a fan of `cliArgs.enc` as a filename—can we make it all lower case? Mixed case filenames seem weird to me.

Reply:
Fixed in d29591d.

Reply:
BTW, I'm not a fan of `filenameswithoutdelimitersofanykindbecauseifindthemhardtoparse.txt`. 😄 So I went with the hyphenated `cli-args.enc`. Hope you're ok with this.

Reply:
Sure, it's fine by me 🙂